docs/application/dotnet/guides/text-input/voice-control-manager.md

```diff
@@ -0,0 +1,1073 @@
+    -   Voice control service state
+        -    The voice control service states are controlled by starting and stopping command recognition.
+    -   Current language
+        -    A command is valid only when the command language is the same as the current language. The current language can be changed by changing the application or display language on the device.
+        -    You can get a notification of the language change in a callback. If the display language is changed to a language that is not supported, the voice control service language changes to the first language that is supported by the voice control engine.
+    -   Supported language
+        -    You can retrieve a list of supported languages to check whether the language that you want is supported.
+
+
+To use the voice control manager, follow these steps:
+
+1.  Create a handle and [register event handlers](#callback).
+
+    The initialization allows the voice control to distinguish your application from any other application that also uses voice control. Only one voice control manager instance can work on the device. Therefore, if other application including voice control manager exists, your application cannot work properly on one same device.
+
+    The registered callbacks can be notified of the changes in service status, current language, recognition results, and errors.
+
+2.  Set commands.
+
+    You can create a command list, and add or remove individual commands in the list. While creating an individual command, set the command text and type for each command handle. When all the commands are created and added to the command list, set the command list to the voice control manager for recognition.
+
+3.  Prepare the voice control manager.
+
+    The preparation connects the voice control service for background work, such as recording and recognition the voice.
+
+    When the application initializes and prepares the voice control manager, the voice control service is invoked and connected for the background work. The service and the application communicate as server and client.
+
+4.  Get the recognized command result and ASR.
+
+    The recognized command result and ASR are sent through a registered callback.
+    You can receive the command matching result that the user uttered.
+    Multiple recognition results can happen if the duplicated commands are registered or you request multiple commands. In this case, you can select or reject the results using the `VoiceControlManagerClient.SetRecognizedCommandsSelectionDelegate()` callback.
+
+5.  When no longer needed, unset the voice control manager.
+
+    You must disconnect the voice control service and deinitialize the voice control manager using the `Unprepare()` and `Deinitialize()` methods of the [Tizen.Uix.VoiceControlManager](https://samsung.github.io/TizenFX/latest/api/Tizen.Uix.VoiceControlManager.html) class.
+
+The figure is the overall configuration for voice control framework :
```

Comment:
@sungraejo 

Global Comment:
Please put a space before starting brackerts in a sentence.

Kindly fix the spacing of colon, brackets, and alignment issues here:
![image](https://user-images.githubusercontent.com/42992085/78654845-ca49d600-78e2-11ea-9f87-e0992f69b95c.png)

Reply:
@rishu-samsung Also please review this additional paragraph related with https://github.com/Samsung/tizen-docs/pull/1048#discussion_r402774882